packages/docs/page-config/ui-elements/button/examples/WithIcon.vue

```diff
@@ -1,29 +1,25 @@
 <template>
-  <div class="flex items-center">
+  <div class="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-4 gap-4">
```

Comment:
Would be nice for these buttons not to be so wide.
<img width="1030" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/23530004/e0258b23-1970-4c25-86f7-496478996aac">

Looks like grid solution doesn't work well on buttons. We need to wrap it with div, maybe.

Reply:
Or we can set `max-width: max-content`.